### PR TITLE
DOC: fix typos (minor)

### PR DESCRIPTION
**Pull request by sciunto**
I discovered few typos...

### CODE REVIEW

**Comment by jni** [comment]
Hi @sciunto! Thanks for spotting these! Actually, I think "extent" should be singular both in these corrections and in the prior sentences. Do you mind fixing them too?

**Comment by sciunto** [comment]
Fixed @jni :)

File: skimage/draw/_draw.pyx [hunk 1]
```diff
@@ -19,7 +19,7 @@ def _coords_inside_image(rr, cc, shape, val=None):
     rr, cc : (N,) ndarray of int
         Indices of pixels.
     shape : tuple
-        Image shape which is used to determine maximum extents of output pixel
+        Image shape which is used to determine the maximum extent of output pixel
```
Comment (JDWarner) [inline skimage/draw/_draw.pyx]: These lines are now all 82 characters long!

Comment (jni) [inline skimage/draw/_draw.pyx]: @JDWarner the wonders of hard-wrapping text... =\

**Comment by jni** [comment]
@sciunto can you fix the line length now? (must fit in 80 characters)

**Comment by sciunto** [comment]
sorry for that.

**Comment by JDWarner** [comment]
Thanks for the fixes, @sciunto! Looks good now.

**Pull request merged**